net/cloak/files/cloak.init

```diff
@@ -0,0 +1,75 @@
+#!/bin/sh /etc/rc.common
+# Copyright (C) 2010-2011 OpenWrt.org
+
+START=90
+
+USE_PROCD=1
+NAME=cloak
```

Comment:
Shouldnt you use rather hard coded path here?

Reply:
What do you mean? `NAME` variable?